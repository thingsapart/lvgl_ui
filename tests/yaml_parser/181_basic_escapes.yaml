# Tests basic C-style escape sequences in double-quoted strings.
message: "Line 1\nLine 2\tIndented\nQuote: \" Backslash: \\ Forward-slash: \/"
