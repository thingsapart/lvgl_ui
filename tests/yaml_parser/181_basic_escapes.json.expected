[{
		"message":	"Line 1\nLine 2\tIndented\nQuote: \" Backslash: \\ Forward-slash: /"
	}]
